modules/io/common/src/test/resources/testdata/twkb/generate_test_data.sql

```diff
@@ -0,0 +1,120 @@
+CREATE OR REPLACE FUNCTION generate_twkb_test_data(
```

Comment:
Would it be possible to document how to use this file?

Reply:
I added some documentation, please let me know if there's other info you would like to know.